chain/cosmos/chain_node.go

```diff
@@ -678,11 +682,6 @@ func (tn *ChainNode) CreateNodeContainer(ctx context.Context) error {
 	return nil
 }
 
-func (tn *ChainNode) StopContainer(ctx context.Context) error {
```

Comment:
Never called.

Reply:
Was also used for the juno halt recovery to test taking down nodes, writing a new genesis file, then bringing them back up. Will be nice for one off tests to still have the ability to do this.